controllers/suite_test.go

```diff
@@ -46,6 +46,8 @@ var (
 )
 
 func TestAPIs(t *testing.T) {
+	SetDefaultEventuallyPollingInterval(50 * time.Millisecond)
+	SetDefaultEventuallyTimeout(30 * time.Second)
```

Comment:
Now that we have defaults, we can remove the explicit timeout from everywhere else in future PRs